cafebabel/templates/users/edit.html

```diff
@@ -43,6 +43,12 @@ <h2>About</h2>
       <label for=about>Bio</label>
       <textarea id=about name=about>{{ user.profile.about }}</textarea>
     </div>
+    {% if current_user.has_role('editor') %}
+    <div class=checkbox>
+      <label for=editor>Editor role</label>
+      <input type=checkbox id=editor name=editor value=1 {% if user.has_role('editor') %}checked{% endif %}>
```

Comment:
Always `value=1`?

Reply:
Yes, that's the value of the checkbox.
If empty, value equals to 'on'
the fact of being checked or not is managed by the `checked` attribute